### ISSUE TITLE
Updated demo to use MSAL 3.1.0

### ISSUE THREAD

**Issue by MattMinke** [davestewart/msal-vue-demo#1]
currently the demo is using version 2.30.0. there have been some major changes to the MSAL-browser library in version 3.1.0

**Comment by davestewart** [comment]
Can you expand on the changes?

Looks like they may be documented here; https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/v2-migration.md

Do you know if it affects the current demo? I no longer use the platform but it would be nice to update the code.

PRs welcome!

**Comment by MattMinke** [comment]
Yes sorry. should have included link to migration. I am going to hold off doing any PRs until I hear back about on the bug I filed with them. Possible they have broken the logoutPopup. 

https://github.com/AzureAD/microsoft-authentication-library-for-js/issues/6508

**Comment by iamexe** [comment]
Hey, first of all @davestewart thank you so much! 
I'm somewhat a beginner and finally managed to implement this in my app.
I added the pull request to add support to 3.1 and fix this issue.

I don't use Typescript so maybe I'll do a PR later to add a 'no-typescript' folder.

Now @MattMinke I have tried with both latest versions of @azure/msal-browser.
I can confirm that before the change it did't work with 3.1 and as you probably observed it did work after:

npm uninstall @azure/msal-browser
npm install @azure/msal-browser@2
